tic_tac_toe/board.py

```diff
@@ -0,0 +1,48 @@
+from tic_tac_toe.views import BoardDisplay
+
+
+class Board:
+    def __init__(self):
+        self.board = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+
+    def __getitem__(self, index):
+        return self.board[index]
+
+    def turn(self):
+        view = BoardDisplay()
+        token = self.current_player()
+
+        print("Please enter 1-9.")
+
+        view.display_board(self.board)
+
+        user_input = int(input()) - 1
+
+        if self.valid_move(user_input):
+            self.move(user_input, token)
+        else:
+            self.turn()
+
+    def move(self, selection, token="X"):
```

Comment:
I've noticed you assign default values to arguments a lot... wondering why you feel it necessary to default `token` to `"X"` here?